R/RenderSql.R

```diff
@@ -46,8 +46,9 @@ NULL
 #' pattern is used to turn on or off blocks of SQL code.} }
 #'
 #'
-#' @param sql   The parameterized SQL
-#' @param ...   Parameter values
+#' @param sql                         The parameterized SQL
+#' @param showParameterWarnings       Should missing parameter warnings be shown?
```

Comment:
Would you mind making some minor changes here:

Change 'showParameterWarnings' into 'warnOnMissingParameters'.

The explanation could be a bit more informative, like: 'Should a warning be raised when parameters provided to this function do not appear in the parameterized SQL that is being rendered?'

These changes should also be applied to the loadRenderTranslateSql function.